UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -782,35 +782,6 @@ class BTPayPalClient_Tests: XCTestCase {
         }
     }
 
-    func testTokenizeVaultAccount_whenPayPalAppApprovalURLMissingBAToken_returnsError() {
```

Comment:
Why is this test removed? If the URL doesn't contain either a `token` or `ba_token`, I thought we still want to surface an error to the merchant and not proceed with opening the URL to the PayPal app

Reply:
ahhh i thought it'd be ok to remove this test since we're longer checking for the existence of `token` or `ba_token` value before we attempt to app-switch since we expect gw to send us the redirectURL which would contain these params

on second thought, i'm open to re-adding this [check](https://github.com/braintree/braintree_ios/blob/67e0b5d6da3c456d845321a7368bbc6455c80980/Sources/BraintreePayPal/BTPayPalClient.swift#L369) and re-adding this test if we feel this may be a safer approach.

thoughts? @jaxdesmarais @saperi22 @scannillo

Reply:
Yeah, I feel like it may be safer to check that one of these exists. We should make a ticket to do the same on Android if we are not doing the same already.

Reply:
updated in 7d5c59d3b6f4f468796d12212dcb6e4558370c8c